examples/SimpleMontage.py

```diff
@@ -0,0 +1,91 @@
+# ==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ==========================================================================*/
+import sys
+import os
+import itkConfig
+itkConfig.LazyLoading = False
+import itk
+
+itk.auto_progress(2)
+
+if len(sys.argv) < 4:
+  print(f"Usage: {sys.argv[0]} <directoryWithInputData> <outputDirectory> <outputFilename>")
+  sys.exit(1)
+
+inputPath = sys.argv[1]
+outputPath = sys.argv[2]
+outFile = sys.argv[3]
+
+if not inputPath.endswith('/') and not inputPath.endswith('\\'):
+  inputPath += '/'
+
+os.chdir(outputPath)
+outFile = os.path.abspath(outFile)
+
+dimension = 2
+
+stageTiles = itk.TileConfiguration[dimension]()
+stageTiles.Parse(inputPath + "TileConfiguration.txt")
+actualTiles = stageTiles # we will update it later
+
+ScalarPixelType = itk.UC
+OriginalPixelType = itk.RGBPixel[itk.UC]
+ScalarImageType = itk.Image[ScalarPixelType,dimension]
+OriginalImageType = itk.Image[OriginalPixelType,dimension]
+
+oImages = [] # RGB images, for mosaic creation
+sImages = [] # grayscale images, for registration
+for t in range(stageTiles.LinearSize()):
+  origin = stageTiles.Tiles[t].Position # AttributeError: 'itkTileConfiguration2' object has no attribute 'Tiles'
```

Comment:
Public member ivars are not exposed in Python. @thewtex is this expected?

Reply:
According [to the documentation](http://swig.org/Doc4.0/SWIGPlus.html#SWIGPlus_member_data), they may be exposed. What is the result of `dir(stageTiles)`?

Reply:
dir(stageTiles)
`['LinearIndexToNDIndex', 'LinearSize', 'Parse', 'TryParse', 'Write', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__swig_destroy__', '__weakref__', 'getNextNonCommentLine', 'nDIndexToLinearIndex', 'parseLine', 'this', 'thisown']`

Reply:
A getter may be required.